pylsp_ruff/ruff_lint.py

```diff
@@ -187,6 +177,8 @@ def build_args(document: Document, options: dict) -> list:
     args = ["--quiet"]
     # Use the json formatting for easier evaluation
     args.extend(["--format=json"])
+    # Do not attempt to fix -> returns file instead of diagnostics
+    args.extend(["--no-fix"])
 
     # Convert per-file-ignores dict to right format
     per_file_ignores = options.pop("per-file-ignores")
```

Comment:
Off-topic: has this been working for you? I haven’t tried it, but it feels tricky to get all the cases right here especially with the need to escape globs and stuff.

Reply:
I haven't come across any issues but it's good to keep that in mind for any future issues :+1: